docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -89,6 +89,26 @@ Python style
         """
         ...
 
+* In many places in Django's codebase, we catch low-level exceptions that have
+  been raised, and raise our own exception instead-- Also known as "wrapping
```

Comment:
space around dashes and lower case - "instead -- also known"

Reply:
Fixed.